dist/ngWebmakerLogin.js

```diff
@@ -1293,13 +1296,16 @@ ngModule.directive('wmJoinWebmaker', [
 
             if (email) {
               $scope.user.email = email;
-              joinController.validateEmail(email);
```

Comment:
I moved it to later in the process, it now validates inside "displayEmailInput" event.

https://github.com/mozilla/webmaker-login-ux/pull/143/files#diff-cedb0953ab67cb6defeeaf13421801a6R1317